.github/publish-maven.sh

```diff
@@ -30,7 +30,7 @@ fi
 cd maven-java
 
 # update poms with the version extracted from sbt dynver
-mvn --quiet --batch-mode versions:set -DnewVersion=${SDK_VERSION}
+mvn --quiet --batch-mode --activate-profiles patch-version versions:set -DnewVersion=${SDK_VERSION}
```

Comment:
active `patch-version` profile to be able to change the version. 

See below

Reply:
Doesn't the `release` profile also need enabling?

Reply:
No, we are only patching the version in the pom. 
Later we run the release and then we activate `release`